### ISSUE TITLE
502 Bad Gateway error

### ISSUE THREAD

**Issue by dennis2k** [jquery/download.jqueryui.com#306]
Having trouble downloading through your bundler. Giving me a bad gateway 502 error.

![capture](https://cloud.githubusercontent.com/assets/9319776/16302590/54b87b5a-394c-11e6-9f92-b78225a9086c.PNG)

**Comment by amouratoglou** [comment]
me too

**Comment by arschmitz** [comment]
I can't reproduce this things seem to be working.

**Comment by aulvi** [comment]
Sorry about the downtime folks. I am aware of the issue and working on a fix right now. Please hang tight while I reboot the server, this will take a few minutes.

**Comment by LooInSpain** [comment]
Problem still seems to there - Jqueryui website working OK but as soon as you try to download you get the cloudflare warning
![jqueryui](https://cloud.githubusercontent.com/assets/20208564/16456645/de41d35a-3e18-11e6-921e-15138d461932.png)

**Comment by aulvi** [comment]
@LooInSpain Sorry you are having trouble.

I cannot reproduce the issue, could you give me more detail around the steps you are taking so I can get it to fail for me as well? Thanks!

**Comment by LooInSpain** [comment]
The download appears to be working now, it was back up and working last 
night

**Comment by d-signet** [comment]
same issue

download link from

http://jqueryui.com/download/#!version=1.9.2&zThemeParams=5d00000100d905000000000000003d8888d844329a8dfe02723de3e5700bbb34ecf36cdef1e1654faa0015427bdb9eb45ebe89aaede0ec5f0d92417fcc6551ce804ff46bf543167700d2cf4e583f2515147aacb86ffa11c0e588dae72a13c98dc37478199f7eea536e99705404149ad3e00bc8b1ab61d369a189b8543dfe7c6f3b8773513a278552790da0900b2f42f0b0e6179039f2e651d5c47e4ac50f07778fc2713d556a4afb61813d5b75a1ec15a2d5212d1f1659a5f057b4052fdb33d69a58aa8d949eab1f67f0c323debde91db3cfb25ffc8d1c7b382957d6119365fef24d89ecddbd6d9ee8b0aa87e14ae04d84fbe2f9f4f985375f6d9c5a7a792464989b8516f8ddf5b42265274e02073d51ceb74c26c68c54ba86a38a5a58cba6b98538775483ffa42ba14c13f6bd3cb3b00b83a1f9798090fa40d23d6e02e63377dc96ee82cb807474fe2039f840c4ab79386167b37f22b39520852efdd07dcf5dbc23043415afd6170681db3d1f2a6a30eebef084ebb302c0003edee2fc07a45705beb66523049c95e6b2b696f80fcbc582686ef24e4b81b1371b70133a71f7122fcd4162b98c05e71e363821fa879278b045062f860659c67c50a761fdfadf640119d9fd07e5dfff3ceae900


not working. 

Bad gateway error

**Comment by sdn2525** [comment]
Still could not download jqueryui.js because of the **download.jquery.ui _Host Error_**

<img width="1056" alt="screen shot 2016-07-08 at 5 18 34 pm" src="https://cloud.githubusercontent.com/assets/3153957/16686583/c8c26b88-4530-11e6-8474-2af0bdabfe8f.png">

**Comment by hh-com** [comment]
Still could not download (502(BadGateway)) ...
http://www.host-tracker.com/InstantCheck/ResultComplete/dd2ab263-9b47-e611-80c1-0003ff733473

**Comment by Lonniebiz** [comment]
Same here. Can't download jqueryui due to this.

**Comment by mcorujao** [comment]
Same problem!
Mesmo problema!

**Comment by dllhell** [comment]
still 502

**Comment by evolucionapp** [comment]
Error 502 when I try to download a custom theme from Themeroller. Just create a custom theme like this one: 

http://jqueryui.com/themeroller/#!zThemeParams=5d00000100c405000000000000003d8888d844329a8dfe02723de3e57019031df7501db103e8db271020b827b7d889996e3970f9990b56d331eb18c5db335e78b77ca80b66848266bc36624000d8c8f2ad350466408f73e6ee62d366fe2d7464a0ae7783a919e3eae3205d1e65d7241471d33ad61f21942b1e3e2c15c52917313900fc176e17d442ffd55e190a22b3a06dd7a91edfe65e0e285d6fef3bca6f5718a24dd83b5419ac572e9671177cd01642ec2e3cb9db812df6e8ab3002ba9965eee7e7fd2d9043d17dab4f4fbda86242ed97bcac6512427ca4fa23a25bf23fcf80fe127b4892e2271a7237ab56a2bbc9494962e21a35b9e25dc034ffc1ce50a30f4d82271577e677dd2475c70f31b420ac636d0d2aaf67cd01b8e5a5bcce516bf837ec51ea60b467209224bceb941ab6091381d798d4f30f51d8cabeacb55caeadbd856c779c7e345d4d3faf6c6463fba2238961cf0e8d9c8ad863555c4fbe34a4d197049baa1c66c67921c83bace190086ec818512a7ad381096bcf0fc12dbc5cc1786bf0ed60e0148d197782ad01d98fc52c994e540ac91af4c1766782d145ebbdd376c49d14ee056b7abec7e361462cb4fccf97c8b2a477d00beaf8ce6f76813e839c77c4d05aee5b1525d23ee443fd4219b4

Then click on Download, and then click on Download again (no modifications in any additional download form). The reported error page is displayed.

**Comment by aulvi** [comment]
Back online (for now).

The builder is a little node.js application and I'm seeing errors in the system log when it crashes, appears to me like one of the node modules isn't happy. I'm going to keep digging.

**Comment by evolucionapp** [comment]
Yes, I was able to download the custom theme I built, a few minutes after I posted my "me too" report. Thanks a lot for your help.

**Comment by 0229xiang** [comment]
Error 502

**Comment by fdeurioste** [comment]
Same here.

**Comment by chiu0602** [comment]
same here

**Comment by TCKay** [comment]
same here

**Comment by orta-sanz** [comment]
Same.

**Comment by HwangJaeYoung** [comment]
I have the same problem such as 502 bad gateway!

**Comment by hastom** [comment]
Error 504

**Comment by brainfoolong** [comment]
First Error 502, now not even connectable to the site.

**Comment by monoregent** [comment]
As i'm having error 504 and i needed jqueryui ASAP, i used the quick downloads option. At least the Stable one is working if this help anyone.(i think there's a grammar error somewhere, sorry if it's true).

**Comment by shnataly** [comment]
Same is there. Error 502: Gateway time-out

**Comment by scottgonzalez** [comment]
I'm locking this issue because all of the me too replies aren't helpful and are flooding our inboxes.
